_posts/en/newsletters/2023-04-12-newsletter.md

```diff
@@ -0,0 +1,184 @@
+    amount and "-50,000 sats" indicates she wants to splice out that
+    amount.  He also mentions, but does not go into detail, about a
+    concern involving zero-conf splicing.
+
+- **Proposed BIP for transaction terminology:** Mark "Murch" Erhardt
+  [posted][erhardt terms] the draft of an [information BIP][terms bip]
+  to the Bitcoin-Dev mailing list that suggests a vocabulary to use for
+  refering to parts of transactions and concepts relating to them.  As
+  of this writing, all replies to the proposal were supportative of the
+  effort.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Don't download witnesses for assumed-valid blocks when running in prune mode][review club 27050]
+is a PR by Niklas Gögge (dergoegge) that improves the performance of Initial Block Download
+(IBD) by not downloading witness data on nodes that are configured to both
+[prune block data][docs pruning] and use [assumevalid][docs assume valid]. This
+optimization was discussed in a recent [stack exchange question][se117057].
+
+{% include functions/details-list.md
+  q0="If assume-valid is enabled but not pruning, why does the node need
+     to download (non-recent) witness data, given that the
+     node won't be checking this data? Should this PR also disable
+     witness download in this non-pruning case?"
+  a0="The witness data is needed because peers may request non-recent
+      blocks from us (we advertise ourselves as non-pruned)."
+  a0link="https://bitcoincore.reviews/27050#l-31"
+
+  q1="How much bandwidth might be saved by this enhancement during an IBD?
+      In other words, what is the cumulative size of all witness data up to
+      a recent block (say, height 781213)?"
+  a1="110.6 GB, which is about 25% of the total block data. One participant
+      pointed out that 110 GB is about 10% of his monthly ISP download
+      limit, so this is a significant reduction. Participants also expected
+      the percentage savings to increase with the recently expanded use of
+      witness data."
+  a1link="https://bitcoincore.reviews/27050#l-52"
+
+  q2="Would this improvement reduce the amount of download data from all
+      blocks back to the genesis block?"
+  a2="No, only since segwit activation (block height 481824); blocks before
+      segwit do not have witness data."
+  a2link="https://bitcoincore.reviews/27050#l-73"
+
+  q3="This PR implements two main changes, one to the block request logic
+      and one to block validation. What are these changes in more detail?"
+  a3="In the block request logic, remove `MSG_WITNESS_FLAG` from the fetch
+      flags so our peers don't send us the witness data. In validation,
+      skip witness merkle tree checks because we don't have the witness."
```

Comment:
I think the order here matters, otherwise the cause and effect relationship isn't quite right (when the logic for skipping merkle tree checks is added, we still have the witness data).
```suggestion
  a3="In validation, when script checks will be skipped, also skip witness merkle tree checks.
      In the block request logic, remove `MSG_WITNESS_FLAG` from the fetch
      flags so our peers don't send us the witness data."
```